ccd-definition/ComplexTypes/1_directionResponse.json

```diff
@@ -0,0 +1,86 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "DirectionResponse",
+    "ListElementCode": "assignee",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "DirectionAssignees",
+    "ElementLabel": "Party",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "DirectionResponse",
+    "ListElementCode": "directionId",
+    "FieldType": "Text",
+    "ElementLabel": "Direction ID",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "assignee=\"DO_NOT_SHOW\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "DirectionResponse",
+    "ListElementCode": "complied",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "YesOrNo",
+    "ElementLabel": "Has this direction been complied with?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "DirectionResponse",
+    "ListElementCode": "file",
+    "FieldType": "Document",
+    "ElementLabel": "Upload file",
+    "HintText": "You can upload JPG, BMP, PNG, TIF or PDF files up to 10MB in size.",
```

Comment:
Do we actually make this restriction? Or is it more of a guideline.

Reply:
I think this is GDS standard for upload files... it was in the design, so I added

Reply:
I think this [confluence](https://tools.hmcts.net/confluence/display/RCCD/Document+Store+-+Global+Whitelist+of+Document+Types+and+Maximum+File+size) page links to the actual restrictions

Reply:
handy to know! 100MB files 😮

Reply:
BIG